src/assets/js/toolbar.js

```diff
@@ -313,14 +313,25 @@
         requestCounter[0].className = className;
     }
 
+    /**
+     * Should AJAX request to be logged in debug panel
+     *
+     * @param requestUrl
+     * @returns {boolean}
+     */
+    function shouldTrackRequest(requestUrl) {
+        var a = document.createElement('a');
```

Comment:
What a weird way to normalize URLs 😄 But should be reliable. Well done 👍

Reply:
At least it should be work everywhere unlike [URL()](https://developer.mozilla.org/ru/docs/Web/API/URL/URL)